test/e2e/system_reset_test.go

```diff
@@ -110,4 +110,24 @@ var _ = Describe("podman system reset", Serial, func() {
 		session2.WaitWithDefaultTimeout()
 		Expect(session2).Should(ExitCleanly())
 	})
+
+	It("system reset - restart socket", func() {
+		SkipIfRemote("system reset not supported on podman --remote")
+		SkipIfNotActive("podman.socket", "podman.socket is not active")
+
+		session := podmanTest.PodmanExitCleanly("info", "--format", "{{.Host.RemoteSocket.Path}}")
+		socketPath := session.OutputToString()
+
+		curlBeforeReset := StartSystemExec("curl", []string{"-s", "--max-time", "3", "--unix-socket", socketPath, "http://localhost/libpod/_ping"})
+		curlBeforeReset.WaitWithDefaultTimeout()
+		Expect(curlBeforeReset).Should(ExitCleanly())
+		Expect(curlBeforeReset.OutputToString()).Should(Equal("OK"))
+
+		podmanTest.PodmanExitCleanly("system", "reset", "--force")
+
+		curlAfterReset := StartSystemExec("curl", []string{"-s", "--max-time", "3", "--unix-socket", socketPath, "http://localhost/libpod/_ping"})
+		curlAfterReset.WaitWithDefaultTimeout()
+		Expect(curlAfterReset).Should(ExitCleanly())
+		Expect(curlAfterReset.OutputToString()).Should(Equal("OK"))
+	})
```

Comment:
This test does not seem useful, it assumes the podman.service works while e2e tests generally should be fully isolated. 
And the socket is disabled in all our CI envs AFAIK so the test is effectively skipped in CI thus there is no coverage either.

I don't think curl is needed either. If we really want to test then it seems enough to test if socket path exist before rest if it does it must exist after reset, so a simple
`Expect(path).To(BeAnExistingFile())` should be enough.

But maybe no test works best. Or if we want a test maybe in test/system as better as we do require full ownership over the storage there.

Reply:
Okay, I didn't know that. I will remove test.